bin/release.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+#
+# Usage: release.sh [--dry-run]
+set -e
+
+if [[ "$1" == "--dry-run" ]]; then
+  DRY_RUN=true
+  NPM_ARGS=" --dry-run"
+  echo "Starting a dry run release..."
+fi
+
+echo "Starting a release..."
+echo " "
+echo "This will:"
+echo "- check that you're logged in to npm as the correct user"
+echo "- publish the package to npmjs.org if it has not been published already"
+echo "- check that there is not already a tag published"
+echo "- create a new tag"
+echo "- push the tag to remote origin"
+echo " "
+
+read -r -p "Do you want to continue? [y/N] " continue_prompt
+
+if [[ $continue_prompt != 'y' ]]; then
+    echo "Cancelling release, if this was a mistake, try again and use 'y' to continue."
+    exit 0
+fi
+
+NPM_USER=$(npm whoami)
+if ! [ "digital-marketplace" == "$NPM_USER" ]; then
+    echo "⚠️   FAILURE: You are not logged in with the correct user."
+    echo "Please login using the details found in https://github.com/alphagov/digitalmarketplace-credential"
+    exit 1
+fi
+
+echo "📦  Publishing package..."
+
+# Try publishing
+cd package
+npm publish $NPM_ARGS
+echo "🗒 Package published!"
+cd ..
```

Comment:
It's often nicer to use `pushd` and `popd` for this kinda stuff...

Reply:
Thanks I've learnt something new 😎, I haven't come across both commands before but I think in this instance I'll still leave it with `cd`